### PR TITLE
API Documentation

### PR DESCRIPTION
**Pull request by akhuoa**

### CODE REVIEW

**Review by agarny — COMMENTED** [review]
A few comments and various edits. Regarding the demo, I am not sure what can be done about it since `SimulationVuer` does indeed require the URL to the location of the SPARC API. @alan-wu, any idea of what can be done about it?

File: docs/.vitepress/config.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+import { defineConfig } from 'vitepress'
+import Components from 'unplugin-vue-components/vite'
+import { ElementPlusResolver } from 'unplugin-vue-components/resolvers'
+
+// Version number
+const versionNumber = process.env.npm_package_version
+
+// https://vitepress.dev/reference/site-config
+export default defineConfig({
+  title: "SimulationVuer Docs",
```
[the remark this answers is not in the document]
Comment (akhuoa) [reply docs/.vitepress/config.js:9]: Yes, this is the format used in other components. @alan-wu, please let me know if you want to change the title format for all components.

File: docs/.vitepress/config.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        items: [
+          { text: 'Live Demo', link: '/demo' },
+          { text: 'API References', link: '/components/SimulationVuer' },
+        ]
+      }
+    ],
+
+    sidebar: [
+      {
+        text: 'Live Demos',
```
[the remark this answers is not in the document]
Comment (akhuoa) [reply docs/.vitepress/config.js:27]: @agarny I will update this.

File: docs/.vitepress/config.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        items: [
+          { text: 'SimulationVuer Demo', link: '/demo' }
+        ]
+      },
+      {
+        text: 'API References',
```
[the remark this answers is not in the document]
Comment (akhuoa) [reply docs/.vitepress/config.js:33]: @agarny I will update this.

File: docs/index.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+layout: home
+
+hero:
+  name: "SimulationVuer"
+  text: "API documentation"
+  tagline: SimulationVuer is a Vue component used in the SPARC portal to run some simulation-based datasets.
+  actions:
+    - theme: brand
+      text: Demo
+      link: /demo
```
[the remark this answers is not in the document]
Comment (akhuoa) [reply docs/index.md:12]: This format is based on the ScaffoldVuer. @alan-wu, please let me know if you want to change this for all components.

File: docs/.vitepress/config.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (akhuoa) [reply docs/.vitepress/config.js]: This is updated in [eca1b46](https://github.com/ABI-Software/simulationvuer/pull/56/commits/eca1b46ef1c9200388db23654e923b46eeee63b1).

File: docs/.vitepress/config.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (akhuoa) [reply docs/.vitepress/config.js]: This is updated in [eca1b46](https://github.com/ABI-Software/simulationvuer/pull/56/commits/eca1b46ef1c9200388db23654e923b46eeee63b1).

File: docs/index.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (akhuoa) [reply docs/index.md]: The "API Reference" button style is changed to the main button style, and the "Demo" button style is the default style.

File: docs/.vitepress/config.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (akhuoa) [reply docs/.vitepress/config.js]: "Docs" is removed from the title.

**Pull request merged**